### PR TITLE
Added class hashes mapping

### PR DESCRIPTION
**Pull request by MaksymilianDemitraszek**

### CODE REVIEW

**Comment by MaksymilianDemitraszek** [comment]
Closes #1675 

## Introduced changes


- Calculates all local class hashes when loading artifacts, then injects corresponding names to call trace. 

## Checklist



- [x] Linked relevant issue
- [ ] Updated relevant documentation
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

**Comment by piotmag769** [comment]
Link correct issue pls (#1593)

**Comment by MaksymilianDemitraszek** [comment]
TODO:
- tests
- parallel class hash calculation

File: crates/forge-runner/src/contracts_data.rs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+
+
+
+
+#[derive(Debug, Clone)]
+pub struct ContractsData {
+    pub contracts: HashMap<String, StarknetContractArtifacts>,
+    pub class_hashes: BiMap<String, ClassHash>,
```
Comment (war-in) [inline crates/forge-runner/src/contracts_data.rs:17]: Wouldn't it be good to merge those two? 🤔

Comment (piotmag769) [reply crates/forge-runner/src/contracts_data.rs:17]: Will open another PR (this one was created by `git spr`), but imo those are like two tables in a database

**Pull request closed without merging**